bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -4,30 +4,35 @@
  */
 
 pub extern crate evmc_sys;
-pub use evmc_sys as ffi;
```

Comment:
This seems to be a trend among `-sys` crates, that's the reason I did it.

Reply:
I will re-introduce it in a bit. I just needed to get rid of it for the moment to organize the visibility layout for the procedural macro.

Reply:
I don't fully understand why it needs to be removed now and be readded later introducing a lot of changes across the way?